### PR TITLE
STDV-840 onboard new container corning-3830 and corning-3764

### PR DESCRIPTION
**Pull request by xiliuStrateos**
@polarpine Do we need to release a new version of the autoprotocol?

### CODE REVIEW

**Comment by xiliuStrateos** [comment]
> @polarpine Do we need to release a new version of the autoprotocol?

Answer my own question, we do. CPMC is pending on the new containers. Lol

File: autoprotocol/container_type.py [hunk 1]
```diff
@@ -1238,6 +1238,84 @@ def row_count(self):
...
+#:
+FLAT384BLACKUBOTTOM = ContainerType(
+    name="384-well Black Clear Round Bottom Ultra-low Attachment Microplate",
+    shortname="384-ubottom-black-clear-tc",
+    is_tube=False,
+    well_coating=None,
+    sterile=True,
+    well_count=384,
+    col_count=24,
+    well_depth_mm=12.55,
```
Comment (aaronstrateos) [inline autoprotocol/container_type.py:1250]: make well_depth_mm a Unit:

well_depth_mm = Unit(12.55, "millimeter"),

File: autoprotocol/container_type.py [hunk 2]
```diff
@@ -1238,6 +1238,84 @@ def row_count(self):
...
+#:
+FLAT384BLACKTC = ContainerType(
+    name="384-well Black Clear Flat Bottom Tissue Culture Microplate",
+    shortname="384-flat-black-clear-tc",
+    is_tube=False,
+    well_coating=None,
+    sterile=True,
+    well_count=384,
+    col_count=24,
+    well_depth_mm=11.55,
```
Comment (aaronstrateos) [inline autoprotocol/container_type.py:1289]: well_depth_mm = Unit(11.55, "millimeter"),

**Review by aaronstrateos — DISMISSED** [review]
Looks good only thing is:
- [ ] well_depth_mm a Unit where commented

File: autoprotocol/container_type.py (hunk 2 above)
Comment (xiliuStrateos) [reply autoprotocol/container_type.py:1289]: It is actually asking a float in the signature. 
https://github.com/autoprotocol/autoprotocol-python/blob/master/autoprotocol/container_type.py#L54

File: autoprotocol/container_type.py (hunk 1 above)
Comment (xiliuStrateos) [reply autoprotocol/container_type.py:1250]: The signature asks a float type. And most of the containers in this module passes a float, only two don't follow the signature.

**Review by aaronstrateos — DISMISSED** [review]
🚀

**Review by aaronstrateos — APPROVED** [review]

**Pull request merged**